chord_metadata_service/ontologies/utils.py

```diff
@@ -0,0 +1,14 @@
+import xmltodict
+
+
+def readXsdSimpleTypeValues(xsd_file_path: str, type_name: str):
```

Comment:
python functions should be snake case

also, add a return type hint here - and use consistent quotes for dict accesses (i prefer double quotes)

return type should probably be `List[str]` (if katsu still supports python 3.8 - also an import from `typing` will be required) or `list[str]` if its python 3.10+